### ISSUE TITLE
about scrollDirection and exiting overflow-y div inside

### ISSUE THREAD

**Issue by chrisvidal** [ashthornton/asscroll#31]
Hi,
thanks for this light scroll library. 
I was wondering how to get the scrollDireciton, to manipulate the header visibility in my case.
Also, I have a div with an overflow-y that does not work anymore when I enable the scroll, is this a bug or expected?
how to handle this case in your opinions?

thanks

**Comment by ashthornton** [comment]
Hi @chrisvidal , you could get the scroll direction like so:

```Javascript
let previousScrollPos = 0

ASScroll.on('scroll', scrollPos => {
    if (scrollPos > previousScrollPos) {
        // scrolled up
    } else {
        // scrolled down
    }
    previousScrollPos = scrollPos
})
```

Could you show me an example of your overflow issue? Using Codepen or similar.

**Comment by chrisvidal** [comment]
Thanks @ashthornton 

here a quick codepen illustrating the issue

https://codepen.io/chrislabz/pen/qBqOqaP

**Comment by ashthornton** [comment]
Note: related to #24

**Comment by ashthornton** [comment]
@chrisvidal I've added the ability to add a class to an element (like your overflow element) that will prevent ASScroll from scrolling the page. It's available in the latest version (1.7.5 - https://github.com/ashthornton/asscroll/commit/69ecd839d2896f06fe865b607c21637d72848c77).

The default class to add is `.asscroll-block` but you can change this in the configuration options.

It essentially adds an event to the element which stops the `wheel` event from propagating. I wanted a seamless way of doing this without getting the user to add classes but the alternatives aren't efficient enough.

Please let me know if this works for you.
